install.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+import tkinter as tk
```

Comment:
We have dependencies with wx, gtk2, gtk3, qt and now tk ? It's not a big deal and it's not really mandatory, but still...

Reply:
I agree it is light, and we may get rid of gtk someday...